.github/workflows/perf_sysbench.yml

```diff
@@ -23,15 +23,27 @@ jobs:
       fail-fast: false
 
     steps:
-      - name: set PATH to GIT of the newer version 2.9.0
-        run: echo "/usr/local/git/bin" | tee -a $GITHUB_PATH
-      - name: cleanup workspace
-        run: docker run -w /source -v ${PWD}:/source -i centos:8 /bin/bash -c "chown -R $(id -u):$(id -g) * .[^.]*"
+#      TODO: migrate to v2 when Git 2.18 or higher would be available
       - uses: actions/checkout@v1
-      - name: test
-        env:
-          BENCH: 'sysbench'
-        uses: ./.github/actions/perf
+        with:
+          submodules: true
+      - uses: actions/checkout@v2.3.4
+        with:
+          path: sysbench
+          repository: tarantool/sysbench
+      - name: tarantool build
+        run: |
+          git submodule update --init --recursive
+          cmake . -DENABLE_DIST=ON -DCMAKE_BUILD_TYPE=RelWithDebInfo && make -j 4
```

Comment:
Why `-j 4`?

Reply:
`make` without argument brings inconsistency.
` If the -j option is given without an argument, make will not limit the number of jobs that can run simultaneously.`